app/src/main/java/edu/uc/brown5a7/BottomsUp/MainActivity.kt

```diff
@@ -0,0 +1,40 @@
+package edu.uc.brown5a7.BottomsUp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.lifecycle.MutableLiveData
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.LayoutManager
+import edu.uc.brown5a7.BottomsUp.dto.Drink
+import edu.uc.brown5a7.BottomsUp.searchAdaptor.DrinkAdapter
+import edu.uc.brown5a7.BottomsUp.service.DrinkService
+import kotlinx.android.synthetic.main.main_fragment.*
+
+//import edu.uc.brown5a7.BottomsUp.searchAdapter.DrinkAdapter
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var viewAdapter: RecyclerView.Adapter<*>
+    private lateinit var viewManager: LayoutManager
+    private lateinit var drinkAdapter: DrinkAdapter
```

Comment:
Moved variable declaration outside of functions and marked it as private so it can be used by both functions.